docs/docs/signing.md

```diff
@@ -79,84 +99,63 @@ const req = {
 const sig = await client.sign(req)
 ```
 
-### 💾 Calldata Decoding
+## 💾 Calldata Decoding
 
-:::note
-All available calldata decoding options will be documented in this section. More may be added as time goes on.
-:::
+> NOTE: All available calldata decoding options will be documented in this section. More may be added as time goes on.
```

Comment:
The previous note syntax: 

```
:::note
...
:::
```

Makes it look like these nice callouts in the docs: https://gridplus.github.io/gridplus-sdk/signing#-calldata-decoding

Reply:
Oooh they weren't rendering on my markdown viewer so I was confused. Will revert

Reply:
[074aaa8](https://github.com/GridPlus/gridplus-sdk/pull/467/commits/074aaa820cb7ae45a3de58c7cbafbbc6c1fb8287)